javascripts/discourse/components/custom-user-palette.gjs

```diff
@@ -1,16 +1,15 @@
 import Component from "@glimmer/component";
-import didInsert from "@ember/render-modifiers/modifiers/did-insert";
+import { action } from "@ember/object";
 import { service } from "@ember/service";
-import { TrackedArray } from "@ember-compat/tracked-built-ins";
+import AsyncContent from "discourse/components/async-content";
 import icon from "discourse/helpers/d-icon";
-import { listColorSchemes } from "discourse/lib/color-scheme-picker";
+import { i18n } from "discourse-i18n";
 import ColorScheme from "admin/models/color-scheme";
```

Comment:
This code path is not loaded for regular users. Gather this data will probably require modifications in `core`